website/cue/reference.cue

```diff
@@ -533,6 +534,18 @@ _values: {
 	syntax: *"literal" | "file_system_path" | "field_path" | "template" | "regex" | "remap_program" | "strftime"
 }
 
+#TypeAsciiChar: {
+	_args: required: bool
+	let Args = _args
+
+	if !Args.required {
+		// `default` sets the default value.
+		default: string | null
```

Comment:
Cuelang's primitive types do not include anything that seems to fit better than `string` here: https://cuelang.org/docs/tutorials/tour/types/types/